proposals/NativeInt.md

```diff
@@ -172,34 +191,29 @@ nint z = d + x; // RuntimeBinderException: '+' cannot be applied 'System.IntPtr'
 
 The only constructor for `nint` or `nuint` is the parameter-less constructor.
 
-The members of `System.IntPtr` and `System.UIntPtr` other than the constructors and operators are available from `nint` and `nuint`.
-For `nint` the following `System.IntPtr` members are available. The members available on `nuint` are similar.
-
+The following members of `System.IntPtr` and `System.UIntPtr` _are explicitly excluded_ from `nint` or `nuint`:
 ```C#
-public static readonly IntPtr Zero;
-public static int Size { get; }
+// constructors
+// arithmetic operators
+// implicit and explicit conversions
+public static readonly IntPtr Zero; // use 0 instead
+public static int Size { get; }     // use sizeof() instead
 public static IntPtr Add(IntPtr pointer, int offset);
 public static IntPtr Subtract(IntPtr pointer, int offset);
-public override bool Equals(object obj);
-public override int GetHashCode();
 public int ToInt32();
 public long ToInt64();
 public void* ToPointer();
-public override string ToString();
-public string ToString(string format);
 ```
 
-Some members above use `IntPtr` in the signature even when used from `nint`.
-
+The remaining members of `System.IntPtr` and `System.UIntPtr` _are implicitly included_ in `nint` and `nuint`. For .NET Framework 4.7.2:
```

Comment:
The implicit list may change tomorrow with https://github.com/dotnet/corefx/issues/20256. If we have a corresponding section for .NET Core, it might be worth updating at that time.